weekly.go

```diff
@@ -150,6 +149,13 @@ func runWeelyCommandFunc(cmd *cobra.Command, args []string) {
 	title := sprint.Name
 	createWeeklyReport(title, body.String())
 
+	pendingIssues := getIssuesForSprintWithFilter(sprint.ID, func(issue *jira.Issue) bool {
+		return issue.Fields.Status.Name != issuesStatusClosed
```

Comment:
Maybe it's better to use resolution, because a job can be deferred or cancelled as well, where its status is not "Job Closed".